src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/SingleShot.cs

```diff
@@ -51,7 +56,135 @@ public SingleShot(string modelAbsPath, TensorsInfo inTensorsInfo, TensorsInfo ou
                 throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, msg);
             }
 
-            CreateSingleShot(modelAbsPath, inTensorsInfo, outTensorsInfo);
+            CreateSingleShot(modelAbsPath, inTensorsInfo, outTensorsInfo, NNFWType.Any, HWType.Any, IsDynamicMode);
+        }
+
+        public SingleShot(string modelAbsPath,
```

Comment:
All public API should have documentation.

Reply:
ACK!